TC Meeting on 2024-08-06 (#107)
OpenRailAssociation/technical-committee: meetings/2024-08-06.md

# TC Meeting 2024-08-06

Present: Cornelius, Florian, Adrian, Frederik, Max, Loic; Excused: Stephanie

## Agenda

* Max absence
* Project onboarding, any news or open questions?
  * liblrs: https://github.com/OpenRailAssociation/technical-committee/issues/103
  * Should we also create issues for the other projects?
* Project meta data, update on publiccode.yml etc. (see also https://github.com/OpenRailAssociation/technical-committee/issues/106)
* 3LH, involvement of OpenRail Association

## Protocol

* Max absence
  * Will be on parental leave for two months starting end of September
  * Tasks to be taken care of
    * Migration of projects
    * Maintenance of code projects
      * Org-manager (https://github.com/OpenRailAssociation/github-org-manager/blob/main/MAINTAINERS.md), Florian will act as co-maintainer
      * https://github.com/OpenRailAssociation/openrail-org-config, org owners can make changes, OSRD team can approve changes on OSRD teams
      * DNS-settings (https://github.com/OpenRailAssociation/openrail-dns), we still need to set up governance documents in this repo. Not a lot of changes expected. Owners of the org can do changes there.

>     * Technical administration (email addresses for admin acounts, credentials), Cornelius and Max will come up with a proposal how to manage this

Reviewer: ```suggestion
    * Technical administration (email addresses for admin accounts, credentials), Cornelius and Max will come up with a proposal how to manage this
```

* Project onboarding, any news or open questions?

>   * Proposal to track all projects through issues, also the ones in progress, we assign a "onboarding mentor" who is taking responsibility to drive the onboarding activities. We reflect that by assigning the person to the issue

Reviewer: ```suggestion
  * Proposal to track all projects through issues, also the ones in progress, we assign an "onboarding mentor" who is taking responsibility to drive the onboarding activities. We reflect that by assigning the person to the issue
```

  * We use a issue template, based on the checklist, to make the creation of these issues easier, Cornelius will take care of it.
  * liblrs onboarding: https://github.com/OpenRailAssociation/technical-committee/issues/103
  * We link the GitHub accounts from the official TC member list to make it easier to find the people

> * Project meta data, update on publiccode.yml etc. (see also https://github.com/OpenRailAssociation/technical-committee/issues/106)

Reviewer: ```suggestion
* Project metadata, update on publiccode.yml etc. (see also https://github.com/OpenRailAssociation/technical-committee/issues/106)
```
https://www.merriam-webster.com/dictionary/metadata

  * For the project display on the web site we can have a list in the web site generation which links to publiccode.yml files. Then at the generation time the data is pulled from the publiccode.yml and put on the web site.
  * We need some extensions, maybe something like the incubation stage. We also have some railway specific vocabulary we want to be used in some fields, such as the stage. We can add them for our own purposes, but would like to discuss this with the maintainers of the publiccode.yml to try to make things part of the official standard instead of deviating without need

>   * Decision: We need a solution for meta data management. We start with using publiccode.yml and see how it works in the use cases we need most urgently, such as the project display on the web site

Reviewer: ```suggestion
  * Decision: We need a solution for metadata management. We start with using publiccode.yml and see how it works in the use cases we need most urgently, such as the project display on the web site
```
https://www.merriam-webster.com/dictionary/metadata
